tests/Microsoft.Graph.Test/Requests/Functional/OneNoteTests.cs

```diff
@@ -0,0 +1,736 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using Async = System.Threading.Tasks;
+
+namespace Microsoft.Graph.Test.Requests.Functional
+{
+    //[Ignore]
+    [TestClass]
+    public class OneNoteTests : GraphTestBase
+    {
+        private OnenotePage testPage;
+        private Notebook testNotebook;
+        private static string firstSectionID;
+
+        [ClassInitialize]
+        public static void GetTestSectionId(TestContext c)
+        {
+            // Get a page of OneNote sections.
+            IOnenoteSectionsCollectionPage sectionPage = graphClient.Me
+                                                                    .Onenote
+                                                                    .Sections
+                                                                    .Request()
+                                                                    .GetAsync()
+                                                                    .Result;
+
+            // Get a handle to the first section.
+            firstSectionID = sectionPage[0].Id;
+
+            //await Async.Task.Delay(5000);
+        }
+
+        public async void TestPageCleanUp()
+        {
+            await graphClient.Me.Onenote.Pages[testPage.Id].Request().DeleteAsync();
+        }
+
+        public async void TestNotebookCleanUp()
+        {
+            await graphClient.Me.Onenote.Pages[testNotebook.Id].Request().DeleteAsync();
```

Comment:
If I'm understanding this correctly, you can delete a notebook by passing it in as a page ID? Is that correct?